core/api/src/servers/graphql-server.ts

```diff
@@ -190,3 +188,45 @@ export const startApolloServer = async ({
     })
   })
 }
+
+const ApolloServerPluginGraphQLQueryComplexity = ({
```

Comment:
I had to add it here because adding it in a different file is creating a circular dependency (really not sure why because it only use external packages)